### PR TITLE
Samsung.Kies version 2.6.4.17113.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98696)

### CODE REVIEW

**Comment by stephengillie** [comment]
This application gives an unexpected error in the sandbox:
image

File: manifests/s/Samsung/Kies/2.6.4.17113.1/Samsung.Kies.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Kies
+PackageVersion: 2.6.4.17113.1
+PackageLocale: en-US
+Publisher: Samsung Electronics
+PackageName: Kies
```
Comment (stephengillie) [inline manifests/s/Samsung/Kies/2.6.4.17113.1/Samsung.Kies.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.samsung.com/
PackageName: Kies
PackageUrl: https://www.samsung.com/levant/support/kies/
```

**Comment by stephengillie** [comment]
> This application gives an unexpected error in the sandbox

The error in the Application Log gives slightly more information:
> Activation context generation failed for "C:\Users\user\AppData\Local\Temp\WinGet\Samsung.Kies.2.6.4.17113.1\KiesSetup.exe". 
> Dependent Assembly Microsoft.VC90.MFC,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8" could not be found. Please use sxstrace.exe for detailed diagnosis.

Internet searches suggest it's an issue with `Microsoft.VCRedist.2008.x64` (`x86`) or similar.

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**